.env_example

```diff
@@ -16,4 +16,12 @@ DEPLOY_COMFYUI_DIR=/home/ComfyUI/web
 EXAMPLE_REPO_PATH=tests-ui/ComfyUI_examples
 
 # Whether to enable minification of the frontend code.
-ENABLE_MINIFY=true
\ No newline at end of file
+ENABLE_MINIFY=true
+
+# ------ ViteJS related env variables ------
+# Note: the VITE_ prefix is required
+
+# Frontend base URL
+# Note: default / is for http://localhost:5173/
+# Example: /foobar/ will be served as http://localhost:5173/foobar/
+VITE_BASE_URL=
```

Comment:
as discussed on slack, anything that requires a vite config change does not work for real deployment cases, as the deployment server infra is not vite, it's prebaked output as a `*.js` and `*.html` dump